ui/webui/test/prepare-updates-img

```diff
@@ -2,15 +2,20 @@
 
 set -e
 
-# makeupdates must be run from the top level of the anaconda source tree
-cd ../../
-
 # FIXME boot.iso on rawhide does not currently contain the new cockpit dependencies
 # This will change once we include this changes upstream and start building boot.iso with the new dependencies
 # Then we can safely remove this workaround
-test ! -e cockpit-ws*.rpm && curl -LO https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-ws-259-1.fc35.x86_64.rpm
-test ! -e cockpit-bridge*.rpm && curl -LO https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-bridge-259-1.fc35.x86_64.rpm
+(
+    # store these separately so we don't need to download them each time
```

Comment:
I'm not sure this is a good idea unless we also add a minimum version check - I would be afraid that we would end up with stale cockpit packages causing issues.